MAINTAINERS.yml

```diff
@@ -659,6 +660,7 @@ Release Notes:
     - jhedberg
     - fabiobaltieri
```

Comment:
@kartben we normally have the release managers here, should I add your name under collaborators so you get cc'd on these automatically?